<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>815</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GestureApp Settings</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout" stretch="1,7,2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnFindDevice">
          <property name="text">
           <string>Find Devices</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnConnectDevice">
          <property name="text">
           <string>Connect to Device</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="tblDevice">
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
        </property>
        <column>
         <property name="text">
          <string>Device Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Addess</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="plainTextEdit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>815</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionSaveSettings"/>
    <addaction name="actionLoadSettings"/>
    <addaction name="separator"/>
    <addaction name="actionExitApp"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <action name="actionSaveSettings">
   <property name="text">
    <string>Save Settings</string>
   </property>
  </action>
  <action name="actionLoadSettings">
   <property name="text">
    <string>Load Settings</string>
   </property>
  </action>
  <action name="actionExitApp">
   <property name="text">
    <string>Exit App</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
